jackson-jq/src/main/java/net/thisptr/jackson/jq/module/Module.java

```diff
@@ -0,0 +1,7 @@
+package net.thisptr.jackson.jq.module;
+
+import net.thisptr.jackson.jq.Function;
+
+public interface Module {
+	Function getFunction(String fname, int nargs);
```

Comment:
i feel like `Map<String, Function> getAllFunctions();` will be needed at some point. should we add it now?